petab/visualize/plot_data_and_simulation.py

```diff
@@ -110,22 +110,22 @@ def plot_data_and_simulation(
                                  exp_conditions)
 
     # import visualization specification, if file was specified
-    if isinstance(vis_spec, str):
-        if vis_spec != '':
+    if vis_spec:
+        if isinstance(vis_spec, str):
```

Comment:
not sure if that check is needed.

Reply:
well, if vis_spec can be None, '' and str, then, I think, it's needed. 
But maybe excluding '' is better